packages/react-router-dom/index.ts

```diff
@@ -1,9 +1,12 @@
-import type { RouterProviderProps } from "react-router/dom";
+import type {
+  HydratedRouterProps,
+  RouterProviderProps,
+} from "react-router/dom";
 import { HydratedRouter, RouterProvider } from "react-router/dom";
 
 // TODO: Confirm if this causes tree-shaking issues and if so, convert to named exports
 export type * from "react-router";
 export * from "react-router";
 
-export type { RouterProviderProps };
+export type { HydratedRouterProps, RouterProviderProps };
```

Comment:
Exported so they get included in the typedoc output as a clickable link in the `HydratedRouter` page